lib/receipt.spec.js

```diff
@@ -420,20 +410,20 @@ describe('Receipt', () => {
             it('can not be de-serialized', () => {
                 expect(() => Receipt.from({}, stubbedProvider)).to.throw();
             });
+        });
 
-            context('a de-serialized incomplete Receipt', () => {
-                [
-                    payload => delete payload.sender.fees.single,
-                    payload => delete payload.sender.balances.current,
-                    payload => delete payload.nonce
-                ].forEach(modifier => {
-                    context('when ' + modifier.toString(), () => {
-                        it('can not be signed', () => {
-                            let modifiedPayload = {...unsignedPayload};
-                            modifier(modifiedPayload);
-                            let incompleteReceipt = Receipt.from(modifiedPayload, stubbedProvider);
-                            expect(incompleteReceipt.sign()).to.be.rejected;
-                        });
+        context('a de-serialized incomplete Receipt', () => {
+            [
+                payload => delete payload.sender.fees.single,
+                payload => delete payload.sender.balances.current,
+                payload => delete payload.nonce
+            ].forEach(modifier => {
+                context('when ' + modifier.toString(), () => {
+                    it('can not be signed', () => {
```

Comment:
Same as above. The subsequent test cases can have an effect on the current test case, since it's relying on a shallow copy of the `unsignedPayload` object.